File: src/main/java/github/cweijan/ultimate/convert/JavaType.java
package github.cweijan.ultimate.convert;

import java.util.Arrays;
import java.util.List;

/**
 * @author cweijan
 * @version 2019/9/5 12:34
 */
public class JavaType {
    public static final String Integer = "java.lang.Integer";
    public static final String Short = "java.lang.Short";
    public static final String Float = "java.lang.Float";
    public static final String Double = "java.lang.Double";
    public static final String Long = "java.lang.Long";
    public static final String Byte = "java.lang.Byte";
    public static final String byteArray = "[B";
    public static final String ByteArray = "[Ljava.lang.Byte;";
    public static final String Character = "java.lang.Character";
    public static final String String = "java.lang.String";
    public static final String Boolean = "java.lang.Boolean";
    public static final List<String> DATE_TYPE= Arrays.asList("java.time.LocalDateTime", "java.util.Date");
    public static final List<String> BYTE_ARRAY_TYPE=Arrays.asList(byteArray, ByteArray);
}

File: src/main/java/github/cweijan/ultimate/convert/JavaType.kt
package github.cweijan.ultimate.convert

object JavaType {
    const val Integer = "java.lang.Integer"
    const val Short = "java.lang.Short"
    const val Float = "java.lang.Float"
    const val Double = "java.lang.Double"
    const val Long = "java.lang.Long"
    const val Byte = "java.lang.Byte"
    const val byteArray = "[B"
    const val ByteArray = "[Ljava.lang.Byte;"
    const val Character = "java.lang.Character"
    const val String = "java.lang.String"
    const val Boolean = "java.lang.Boolean"
    val DATE_TYPE: MutableList<String> = mutableListOf("java.time.LocalDateTime", "java.util.Date")
    val BYTE_ARRAY_TYPE: MutableList<String> = mutableListOf(byteArray, ByteArray)
}